src/opnsense/mvc/app/controllers/OPNsense/IDS/forms/generalSettings.xml

```diff
@@ -81,4 +102,4 @@
         <help>Send package payload to the log for further analyses.</help>
         <advanced>true</advanced>
     </field>
-</form>
+</form>
```

Comment:
```suggestion
</form>
```

Reply:
Sry, I don't get it. Whats the suggested change here?

Reply:
To revert your unnecessary line ending change at the end of the file. But GitHub seems to be relatively silly regarding this...

Reply:
Ok, GitHub is buggy.

Reply:
No, this is blank line is intentional but the editor seems to think otherwise,

Reply:
There we go. Got it.

Reply:
it's simple unix style to be able to read/concat multiple files without breaking the contents:

    % cat file.has.EOL
    This is a test
    %

vs.

    % cat file.no.EOL
    This is a test%

The reasoning is: every line has a newline, not just all of them but the last one. :)